lib/src/ui/pages/settings/settings_page.dart

```diff
@@ -18,54 +20,72 @@ abstract class _Constants {
   static const sectionItemLeftSpacing = 30.0;
   static const sectionItemTopSpacing = 12.0;
   static const switchLeftSpacing = 10.0;
-  static const backButtonPadding = 8.0;
 }
 
 class SettingsPage extends StatefulWidget {
+  final int selectedHomePage;
+
+  const SettingsPage({this.selectedHomePage});
+
   @override
   _SettingsPageState createState() => _SettingsPageState();
 }
 
 class _SettingsPageState extends State<SettingsPage> {
+  final SettingsNotifier _settingsNotifier = inject<SettingsNotifier>();
```

Comment:
zauważyłem, że łatwiej jest zarządzać stanem, kiedy każdy widok ma swojego notifiera inicjalizowanego w swojej klasie. Dzięki temu kiedy jakiś widok jest odpalany z kilku miejsc to nie trzeba się martwić o to czy jest notifier providowany przy nawigacji do takiego widoku + scope notifiera jest zawsze optymalny